app/components/db_model.py

```diff
@@ -16,6 +16,4 @@
 from sqlalchemy import MetaData
 from sqlalchemy.orm import declarative_base
 
-from app.config import ConfigClass
-
-DBModel = declarative_base(metadata=MetaData(schema=ConfigClass.RDS_SCHEMA_DEFAULT))
+DBModel = declarative_base(metadata=MetaData())
```

Comment:
`metadata=MetaData()` can be omitted  since it doesn't have any additional configuration.
```suggestion
DBModel = declarative_base()
```